Rendera/include/scene.hpp

```diff
@@ -85,29 +90,34 @@ class Scene{
 
         }
         ~Scene(){
```

Comment:
We don't a manually defined destructor because we are removing the array of `HitInfo` structs.